### PR TITLE
Drop support for Ruby 2.5 and 2.6

### PR DESCRIPTION
**Pull request by andyw8**
@jake-olney encountered an issue with RuboCop:

https://github.com/Shopify/liquid/pull/1576

Upgrading rubocop-shopify solves it but it is not compatible with older Rubies.

@macournoyer suggested we should drop these as both are EOL: https://endoflife.date/ruby

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,8 +18,7 @@ group :benchmark, :test do
 end
 
 group :test do
-  gem 'rubocop', '~> 1.4', require: false
```
Comment (andyw8) [inline Gemfile:21]: I removed this since `rubocop` is already a dependency of `rubocop-shopify`.

**Review by macournoyer — APPROVED** [review]
I see no reasons to keep < 2.7 support around. @dylanahsmith do you know if we're using this anywhere w/ Ruby < 2.7?

File: History.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## 5.4.0 (unreleased)
 
+### Breaking Changes
+* Require Ruby >= 2.7 (#1578) [Andy Waite]
```
Comment (andyw8) [inline History.md:6]: Arguably this should be a major release (6.0), depending on how strict we're being with semver.

Comment (dylanahsmith) [reply History.md:6]: There is already a "Changed" section with a potentially breaking change, so this should probably be combined with that section.

**Comment by dylanahsmith** [comment]
@macournoyer it shouldn't be a problem for us.  It could affect users of the system version of ruby.

File: History.md (hunk 2 above)
Comment (andyw8) [reply History.md:6]: I can move it, but I noticed the previous Ruby bump was listed under that section name:

```
### Breaking Changes
* Require Ruby >= 2.5 (#1131, #1310) [Mike Angell, Dylan Thacker-Smith]
```

Comment (andyw8) [reply History.md]: 👍 Updated in 3bbb7aa7baba7602caa95292d809c3fbfe9d57c5

**Review by dylanahsmith — APPROVED** [review]

**Pull request merged**